AbletonMCP_Remote_Script/handlers/automation.py

```
@@ -0,0 +1,424 @@
+            )
+
+        arr_clips = list(track.arrangement_clips())
+        if not arr_clips:
+            raise Exception(
+                "No arrangement clips on track {0} — record or place a clip first".format(track_index)
+            )
+
+        # Determine the time span the caller wants to automate
+        times = [float(p["time"]) for p in automation_points]
+        t_min = min(times) if times else 0.0
+        t_max = max(times) if times else 0.0
```

Ruff (0.15.0)

[error] Local variable `t_max` is assigned to but never used

Remove assignment to unused variable `t_max`

(F841)